tests/e2e/data/observatory/catalog/test_download.py

```diff
@@ -75,6 +75,7 @@ def test_dataset_to_csv_private(self):
     def test_dataset_to_dataframe_public(self):
         df = public_dataset.to_dataframe(self.credentials, limit=PUBLIC_LIMIT)
         assert isinstance(df, GeoDataFrame)
+        assert df.head().geom.type == 'Polygon'
```

Comment:
I think you could access directly the column `assert df.geom.type == 'Polygon'`

Reply:
Yes, you are right, changing it :+1: